.circleci/config.yml

```diff
@@ -43,19 +43,30 @@ jobs:
     docker:
       - image: circleci/node:10
     steps:
+      - checkout
+      - restore_cache: *restore-deps-cache
+      - run: npm install
       - run: npx semantic-release
+      - save_cache: *save-deps-cache
 
   release-web:
     <<: *defaults
     docker:
       - image: circleci/python:2.7-jessie
     steps:
+      - checkout
       - run:
           name: Install awscli
           command: sudo pip install awscli
       - run:
           name: Deploy to S3
           command: aws s3 sync docs/.vuepress/dist/ s3://open-rpc.org/ --delete
+      - run:
+          name: ls them thangs
+          command: ls docs/.vuepress/dist
```

Comment:
Im not convinced that this will be there, dispite copying most of this from their docs